noah/src/main/scala/edu/uci/ics/cloudberry/noah/kafka/package.scala

```diff
@@ -0,0 +1,40 @@
+package edu.uci.ics.cloudberry.noah
+
+import java.io.File
+import java.util.Properties
+
+import com.typesafe.config.ConfigFactory
+import edu.uci.ics.cloudberry.noah.feed.Config
+import org.apache.kafka.clients.producer._
+
+/**
+  * Created by Monique on 7/18/2016.
+  */
+object ProducerKafka {
```

Comment:
is it the general purpose object? if that is the case, it's better to make it as a `class` which take the `config` as a member. the `store` part is more concrete. At the same time as a general purpose class it can be extended.